genomes/models/genome.py

```diff
@@ -0,0 +1,226 @@
+from django.db import models
+
+from analyses.base_models.with_downloads_models import WithDownloadsModel
+from analyses.models import Assembly, Biome
+
+# Annotation field constants
+COG_CATEGORIES = "cog_categories"
+KEGG_CLASSES = "kegg_classes"
+KEGG_MODULES = "kegg_modules"
+ANTISMASH_GENE_CLUSTERS = "antismash_gene_clusters"
+
+
+def default_annotations():
+    return {
+        COG_CATEGORIES: [],
+        KEGG_CLASSES: [],
+        KEGG_MODULES: [],
+        ANTISMASH_GENE_CLUSTERS: [],
+    }
+
+
+class GenomeManagerDeferringAnnotations(models.Manager):
+    """
+    The annotations field is a potentially large JSONB field.
+    Defer it by default, since most queries don't need to transfer this large dataset.
+    """
+
+    def get_queryset(self):
+        return super().get_queryset().defer("annotations")
+
+
+class GenomeManagerIncludingAnnotations(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset()
+
+
+class Genome(WithDownloadsModel):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
These should be in an abstract timestamped model

Reply:
@mberacochea  this has also been implemented in a different PR